src/pages/docs/tutorials/tutorial-kitten-browser/app-setup/index.md

```diff
@@ -139,6 +142,25 @@ npm run serve
 ![Kitten Browser Step 1](KittenBrowser-Step1.png)
 > Tell the kids to avert their eyes!
 
+### ESLint Hints
+
+When you enter the above command, you'll see the compilation warning message below. Let's take a look at what this message means and try to resolve it briefly.
+```console
+Compiled with warnings.
+
+./src/App/App.js
+  Line 13:6:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
+
+Search for the keywords to learn more about each warning.
+To ignore, add // eslint-disable-next-line to the line before.
+```
+The `npm run serve` compiles the source code and creates a web server. In the process of compiling the code, it uses ESLint to perform a static analysis of the code written in JavaScript, JSX. ESLint helps you to find syntax errors, anti-patterns and to write the source code in a consistent code style. For more detail is [about ESLint](https://eslint.org/docs/about/). If you only want a static analysis of the code you wrote, use `npm run lint`.
+
+Back in the meaning of the message, ESLint says img elements should have an `alt` property. That's right. When you use the `<img />` tag, it is recommended to define an `alt` property for accessibility failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img). We'll add the `alt` property to the tag as follows. See if it is compiled successfully.
```

Comment:
```suggestion
Returning to the warning message, ESLint says img elements should have an `alt` property. That's right. When you use the `<img />` tag, it is recommended to define an `alt` property for accessibility failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img). We'll add the `alt` property to the tag as follows. See if it is built successfully.
```

Reply:
"Returning to the warning message" looks better than previous one.
Thank you.
It will be fixed with your suggestion.